M2/Macaulay2/packages/Macaulay2Doc/doc12.m2

```diff
@@ -514,6 +514,30 @@ document {
      "The return value is ", TO "null", "."
      }
 
+doc ///
+  Key
+    printerr
+  Headline
+    print something to stderr
+  Usage
+    printerr x
+  Inputs
+    x:String -- or @ofClass {Net,BasicList}@
```

Comment:
You can also use:
```m2
  Inputs
    x:{String,Net,BasicList}
```

Reply:
Oh cool!  I was wondering if there was a slicker way of doing this.  Thanks!